misk/web/tabs/loader/src/containers/index.ts

```diff
@@ -1,3 +1,2 @@
-import { NavContainer } from "./NavContainer"
-import TabContainer from "./TabContainer"
-export { NavContainer, TabContainer }
\ No newline at end of file
+import LoaderContainer from "./LoaderContainer"
+export { LoaderContainer }
```

Comment:
Could be `export { default as LoaderContainer } from "./LoaderContainer";`.

Reply:
thanks! Will fix this syntax across the code base